modules/gptPreAuction.js

```diff
@@ -190,9 +194,9 @@ export const makeBidRequestsHook = (fn, adUnits, ...args) => {
       let adserverSlot = deepAccess(context, 'data.adserver.adslot');
       let result;
       if (customPreAuction) {
-        result = customPreAuction(adUnit, adserverSlot);
+        result = customPreAuction(adUnit, adserverSlot, adUnitPaths?.[adUnit.code]);
```

Comment:
The conditional chaining was missing before, right?

Reply:
Correct. Without it, https://github.com/prebid/Prebid.js/blob/91587a19096dc09f9391c3cc5b34879c2d2f8475/modules/gptPreAuction.js#L70 would silently break auctions.